src/components/activity/ActivityTypesPanel.svelte

```diff
@@ -15,19 +16,27 @@
   export let gridSection: ViewGridSection;
   export let user: User | null;
 
+  const permissionError = 'You do not have permission to add an activity.';
+
   let filterText: string = '';
+  let hasPermission: boolean = false;
 
+  $: if ($plan !== null) {
+    hasPermission = featurePermissions.activityDirective.canCreate(user, $plan);
+  }
   $: filteredActivityTypes = ($activityTypes ?? []).filter(({ name }) =>
```

Comment:
One funky case here is when you've filtered and see the "no matching activity types found" card which is not disabled (since it isn't interactive) but has an empty tooltip 
<img width="292" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/e2d09149-a238-41d7-8356-739215a02c99">